buildSrc/src/main/java/Dependencies.kt

```diff
@@ -20,6 +20,8 @@ object Versions {
     val kermit = "0.1.5"
     val lifecycle = "2.1.0"
     val karmok = "0.1.7"
+    val ktlint_gradle_plugin = "9.2.1"
+    val robolectric = "4.3"
```

Comment:
4.3.1 exists. I think there was a warning for this but lint now fails to detect it since it's in a separate constant.

Reply:
yup, this was me just cleaning up while I fixed the warning but then forgetting to fix the warning (because like said it got masked)